<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Disconnect_Device">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭设备串口</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Serial_Port">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Serial_Baurdrate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Serial_Parity">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Serial_Data_Bit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Serial_Stop_Bit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Connect_Device">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接设备串口</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>511</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLabel" name="light_sensor_value">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>光强值:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Clear_Plot">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除曲线显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
